script.skinshortcuts/resources/lib/gui.py

```diff
@@ -0,0 +1,2417 @@
+# coding=utf-8
+import os, sys, datetime, unicodedata
+import xbmc, xbmcgui, xbmcvfs
+import xml.etree.ElementTree as xmltree
+from xml.dom.minidom import parse
+from xml.sax.saxutils import escape as escapeXML
+from traceback import print_exc
+import calendar
+from time import gmtime, strftime
+import random
+import json as simplejson
+import urllib.request, urllib.parse, urllib.error
+import _thread as thread
+
+import datafunctions
+DATA = datafunctions.DataFunctions()
+
+import library
+LIBRARY = library.LibraryFunctions()
+
+ADDON        = sys.modules[ "__main__" ].ADDON
```

Comment:
move all this to a common file and get values from there

Reply:
Can you please clarify this?